### ISSUE TITLE
Deprecation warning: appgatesdp_appliance hostname is deprecated as of 5.4

### ISSUE THREAD

**Issue by muram** [appgate/terraform-provider-appgatesdp#278]
The appgatesdp provider displays warning about the appliance's hostname argument is now deprecated as of 5.4, however, removing the `hostname` argument results in an error:

```sh
Error: Missing required argument
│ 
│   on gateways/appgate.tf line 165, in resource "appgatesdp_appliance" "gateway_1":
│  165: resource "appgatesdp_appliance" "gateway_1" {
│ 
│ The argument "hostname" is required, but no definition was found.
```

### Terraform CLI, Terraform appgatesdp Provider Version, and Appgate Appliance version(s)

terraform cli: v1.3.9
terraform-provider-appgatesdp:  v1.0.4
appgate appliance version: 6.1.1

### Affected Resource(s)

* appgatesdp_appliance

### Terraform Configuration Files

Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.

```hcl
resource "appgatesdp_appliance" "gateway_1" {

  name = "gateway-01"
  # hostname = "gateway-01.example.com"
...
```

### Debug Output

N/A

### Panic Output

N/A

### Expected Behavior

Nor error in the output related to missing `hostname` argument

### Actual Behavior

Error: Missing required argument

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

N/A

### References

* #0000

**Comment by danijeel** [comment]
there was plans before to remove this attribute, but things has changed and we still need it as a Required field. We will remove the deprecation notice in the next release https://github.com/appgate/terraform-provider-appgatesdp/pull/282

**Comment by danijeel** [comment]
Closing this issue since this is fixed in [1.1.0](https://registry.terraform.io/providers/appgate/appgatesdp/1.1.0)
